src/components/TextInput/TextInput.jsx

```diff
@@ -3,9 +3,10 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Cleave from 'cleave.js/react';
 import * as InputMasks from './TextInputMasks';
-import './TextInput.scss';
+import styles from './TextInput.module.scss';
 import FormLabel from '../FormLabel/FormLabel';
 import InputValidationMessage from '../InputValidationMessage/InputValidationMessage';
+import '../../styles/utilities.scss';
```

Comment:
Once again, this seems really out of order. See comment on Button.jsx.

Reply:
https://eslint.org/docs/rules/sort-imports could be used.

Here's the order rule I'm using for the home app:

`"import/order": [
      "error",
      { "groups": ["object", "builtin", "external", "internal", "parent", "sibling", "index"] }
    ],`